README.md

```diff
@@ -287,14 +281,7 @@ Documentation for Snap will be kept in this repository for now with an emphasis
 * [plugin signing](docs/PLUGIN_SIGNING.md)
 * [tribe](docs/TRIBE.md)
 
-
-Here are blog posts related to Snap by the team:
-
-* [What if collecting data center telemetry was a snap?](http://nickapedia.com/2015/12/02/what-if-collecting-data-center-telemetry-was-a-snap/) by @lynxbat
-* [My How-to for the Snap Telemetry Framework](https://medium.com/intel-sdi/my-how-to-for-the-snap-telemetry-framework-e3bb641bc740#.6f5nk543t) by @mjbrender
-* [Snap's first GUI is Grafana!](https://medium.com/intel-sdi/snap-first-gui-is-grafana-40bb92df2660#.rgsnfx3w1) by @mjbrender
-* [Adding a global configuration to Snap](https://medium.com/intel-sdi/adding-a-global-configuration-to-snap-b95d8fd8b5e0#.hq3cqgt6t) by @tjmcs1
-
+The blog for the Snap team is here: [intelsdi-x](https://medium.com/intel-sdi). A good first post to read is [My How-to for the Snap Telemetry Framework](https://medium.com/intel-sdi/my-how-to-for-the-snap-telemetry-framework-e3bb641bc740#.6f5nk543t) by @mjbrender.
```

Comment:
Possible rewording of first sentence:
"To learn more about Snap and how others are using it, check out our blog [here](https://medium.com/intel-sdi)"